contents/customers/octomind.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'How Octomind’s marketing team uses PostHog with ease'
+customer: Octomind
+logo: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/octomind_logo_673e0ed777.png
+logoDark: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/octomind_logo_dark_a89deeee90.png
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/adauris_62af5ac994.png
+featuredCustomer: false
+industries:
+  - 'Devtool, SaaS'
+users:
+  - Marketing
+toolsUsed:
+  - Experiments
+  - Surveys
+  - PostHog Cloud
+  - Insights
+  - Web analytics
+  - Product analytics
+date: 2024-11-10
+---
+
+Popular with fast-moving builders on [ProductHunt](https://www.producthunt.com/products/octomind), [Octomind](https://www.octomind.dev/) is an AI-powered end-to-end testing tool for developers. Unsurprisingly for a DevTool company, the team is engineer-led — to the point that even Maria Zahorcova, Octomind's Chief Marketing Officer, has a strong grasp of developer tools like Git. When it came to choosing an analytics tool, however, she was starting from scratch.
+
+“I’m far from an engineer myself, though I’ve always been on the technical side,” says Maria. “I’m also the only marketer at Octomind and I’m very new to analytics. Like a lot of startup marketers I’d only really used Google Analytics before.”
```

Comment:
I'd reformulate this. I used tons of analytics and other tools before I started to work for startups. 

I'm the only marketer at Octomind. I used Google Analytics in startups I worked before.

Reply:
```suggestion
“I’m far from an engineer myself, though I’ve always been on the technical side,” says Maria. “I’m also the only marketer at Octomind. Like a lot of startup marketers I’d used Google Analytics in other teams before.”
```